.circleci/config.yml

```diff
@@ -141,6 +141,7 @@ jobs:
     environment:
       TESTING_PROFILE: non-confidential-tests
       MAVEN_GOAL: verify
+      DOCKER_BUILDKIT: 0
```

Comment:
Can probably put this back to normal if this is disabling the new Docker backend